src/renderer/pages/Staking/Operations/components/Confirmation/Confirmation.tsx

```diff
@@ -86,6 +87,9 @@ export const Confirmation = ({
   const selectedValidatorsAddress = validators?.map((validator) => validator.address);
   const notSelectedValidators = allValidators.filter((v) => !selectedValidatorsAddress?.includes(v.address));
 
+  const signerWallet = signer && wallets.find((w) => w.id === signer.walletId);
+  const walletType = signerWallet?.type || activeWallet?.type;
```

Comment:
Let's put all available values in one place
`const walletType = signerWallet?.type || activeWallet?.type || WalletType.POLKADOT_VAULT;`